sosialhjelp-common-api/src/main/kotlin/no/nav/sosialhjelp/api/fiks/FiksDigisosApi.kt

```diff
@@ -41,7 +41,13 @@ data class Ettersendelse(
 data class DigisosSoker(
     val metadata: String,
     val dokumenter: List<DokumentInfo>,
-    val timestampSistOppdatert: Long
+    val timestampSistOppdatert: Long,
+    val avsender: Avsender?
```

Comment:
Sender KS med avsender, eller er det kun for intern-bruk? kunne ikke se det i swagger-doc'en dems - https://editor.swagger.io/?url=https://ks-no.github.io/api/digisos-api-v1.json

Reply:
Vi får det fra FIKS på JsonDigisosSoker. Andre steder det heller burde ligge? Dette er nå i utgangspunktet kun tenkt for å kunne legge på fagsystem i mock-alt-api.

Reply:
Aha, nei hvis vi får det fra Fiks er det bare å flette inn tenker jeg 👍

Reply:
Eller blander jeg nå? JsonDigisosSoker får vi vel etter vi har hentet dokumentinfo fra metadata-iden?